FND/Accounting/ISO4217-CurrencyCodes.rdf

```diff
@@ -505,6 +498,7 @@
 		<rdf:type rdf:resource="&fibo-fnd-acc-cur;Currency"/>
 		<rdfs:label>Bolívar Soberano</rdfs:label>
 		<skos:definition xml:lang="en">the currency Bolívar Soberano</skos:definition>
+		<skos:note xml:lang="en">The Bolívar Soberano (VES) is redenominated by removing six zeros from the denominations. A new currency code VED/926 representing the new valuation (1,000,000 times old VES/928) is introduced on 1 October 2021 for any internal needs during the redenomination process, but is not replacing VES as the official currency code. The Central Bank of Venezuela will not adopt the new codes in the local system, VES/928 remains in use. The actual currency code VES/928 remains the valid code after 1 October 2021 to use in any future transactions to indicate the redenominated Bolívar Soberano.</skos:note>
 		<fibo-fnd-acc-cur:hasMinorUnit>2</fibo-fnd-acc-cur:hasMinorUnit>
 		<fibo-fnd-acc-cur:hasNumericCode>926</fibo-fnd-acc-cur:hasNumericCode>
```

Comment:
2 numeric codes does not seem right for the same currency - I think the issue is having the URI based on the name - we need a new URI for the old currency

Reply:
No, it is a single currency that has two codes. The  way the standard describes these and the way that the change appears in the TC 68 documentation is that there are two codes, same currency, and the comment explicitly states that this interim thing is not replacing the code, which is still valid. The only thing that changed in this interim window is that the currency was redenominated. It's still considered the same currency. The code list includes two codes, one currency with a single name, in other words.